### DISCUSSION TITLE
Documentation on M1 and UE5

### DISCUSSION THREAD

**Discussion by rcaridade145** [KhronosGroup/MoltenVK#1867] [General]
I came across some projects i wasn't aware of while looking at UE5 and Apple Silicon - https://www.reddit.com/r/unrealengine/comments/12jx9rq/experimental_nanite_support_added_for_m2_based/ - 

Wonder if they are of any interest to MoltenVK @billhollings 

https://github.com/philipturner/metal-float64
https://github.com/philipturner/metal-benchmarks

**Comment by billhollings** [comment]
Thanks for the research! That's very interesting!

**Comment by rcaridade145** [comment]
Does the mean that https://github.com/KhronosGroup/MoltenVK/issues/1692 should work on M2 but not on M1?
